### ISSUE TITLE
fd_limit in the check_arakoon_fd healthcheck

### ISSUE THREAD

**Issue by matthiasdeblock** [openvstorage/openvstorage-health-check#442]
On the Nuvolat and GIG environments we receive a lot of warning messages regarding the filedescriptors for the arakoon clusters:
```
[WARNING] Number of TCP connections exceeded the 95% warning threshold for Arakoon cluster cachebackend03-nsm_06 on node 10.210.3.61, (155/30)
```

I've noticed that the limit was set to 30. Is there any reason why the limit was set to 30?
```
https://github.com/openvstorage/openvstorage-health-check/blob/e4854917c2f00792316bb39f3383843deda829be/ovs/extensions/healthcheck/arakoon_hc.py#L471
```

For example, if you have 2 proxies per node in a cluster of 4 volumedrivernodes that are pointing to that arakoon cluster, you already have a possible connection size of 80 as all the proxies have a max connection pool size of 10.

**Comment by wimpers** [comment]
Maybe something we can pickup with https://github.com/openvstorage/openvstorage-health-check/issues/422

**Comment by JeffreyDevloo** [comment]
Fixed by #447
Packaged in: openvstorage-health-check_3.7.0-dev.1525250711.8ba3b6b-1_amd64.deb

Usage:
```
Usage: healthcheck_cli.py arakoon file-descriptors-test [OPTIONS]

  Verify the number of File Descriptors on every Arakoon does not exceed the
  limit

Options:
  --fd-limit INTEGER  Threshold for the number number of tcp connections for
                      which to start logging warnings
  --help              Show this message and exit.
root@jef-node01:~# 

```

**Comment by JeffreyDevloo** [comment]
Released in https://github.com/openvstorage/openvstorage-health-check/releases/tag/3.7.0
Packaged in openvstorage-health-check_3.7.0-1_amd64.deb
